src/pages/components/CertificateInfo.qml

```diff
@@ -103,146 +105,69 @@ SilicaFlickable {
-                    }
-                }
-                return ""
-            }
-            color: Theme.errorColor
-            opacity: Theme.opacityHigh
-        }
-
-        SectionHeader {
-            //: Header separating out the main security info from the details
-            //% "Certificate"
-            text: qsTrId("sailfish_browser-sh-cert_details")
-            visible: security && !security.certIsNull
-        }
-    }
-
-    Column {
-        id: keyValuePairs
-        width: parent.width
-        spacing: Theme.paddingSmall
-        anchors {
-            top: certInfoColumn.bottom
-            topMargin: Theme.paddingSmall
-        }
-
-        onHeightChanged: calculatedHeight = height
-
-        CertificateLabel {
-            //: Label for the owner field of a TLS certificate
-            //% "Owner"
-            key: qsTrId("sailfish_browser-la-cert_owner")
-            value: security ? security.subjectOrganization : ""
-            valueColor: _secure ? palette.colorScheme === Theme.LightOnDark
-                                  ? "#22ff22" : "#007700" : Theme.secondaryHighlightColor
-            keyWidth: root.keyWidth
+            color:  Theme.secondaryHighlightColor
+            visible: !_secure
+            //% "Do not enter personal data, passwords, card details on this site"
+            text: qsTrId("sailfish_browser-sh-do_not-enter_personal_data")
         }
 
-        CertificateLabel {
-            //: Label for the subject field of a TLS certificate
-            //% "Website"
-            key: qsTrId("sailfish_browser-la-cert_subject")
-            value: security ? security.subjectDisplayName : ""
-            keyWidth: root.keyWidth
-        }
-
-        CertificateLabel {
-            //: Label for the issuer field of a TLS certificate
-            //% "Issuer"
-            key: qsTrId("sailfish_browser-la-cert_issuer")
-            value: security ? security.issuerDisplayName : ""
-            keyWidth: root.keyWidth
+        Button {
+            anchors.horizontalCenter: parent.horizontalCenter
+            //% "Certificate details"
+            text: qsTrId("sailfish_browser-sh-certificate_details")
+            enabled: security && !security.certIsNull
+            visible: enabled
+            onClicked: showCertDetail()
         }
 
-        CertificateLabel {
-            //: Label for the TLS certificate validity period (dates)
-            //% "Valid"
-            key: qsTrId("sailfish_browser-la-cert_valid_dates")
-            value: insecure ? Format.formatDate(security.effectiveDate, Formatter.DateMedium)
-                            + " — "
-                            + Format.formatDate(security.expiryDate, Formatter.DateMedium)
-                          : ""
-            insecure: security && !security.certIsNull && security.notValidAtThisTime
-            keyWidth: root.keyWidth
-        }
+        Column {
+            width: parent.width
```

Comment:
This looks cleaner if the column has the same spacing as the parent column (i.e. `Theme.paddingMedium`). If the ideas is to group the "Current permissions" title and the permission icons together, then I think extra space should be added above the title.